polkadot/zombienet_tests/functional/0012-elastic-scaling-mvp.zndsl

```diff
@@ -0,0 +1,28 @@
+Description: Test that a paraid acquiring multiple cores does not brick itself if ElasticScalingMVP feature is enabled
+Network: ./0012-elastic-scaling-mvp.toml
+Creds: config
+
+# Check authority status.
+validator: reports node_roles is 4
+
+validator: reports substrate_block_height{status="finalized"} is at least 10 within 100 seconds
+
+# Ensure parachain was able to make progress.
+validator: parachain 2000 block height is at least 10 within 200 seconds
+
+# Register the second core assigned to this parachain.
+alice: js-script ./0012-register-para.js return is 0 within 600 seconds
+
+validator: reports substrate_block_height{status="finalized"} is at least 35 within 100 seconds
+
+# Parachain will now be stalled
+validator: parachain 2000 block height is lower than 20 within 300 seconds
+
+# Enable the ElasticScalingMVP node feature.
+alice: js-script ./0012-enable-node-feature.js with "1" return is 0 within 600 seconds
+
+# Wait two sessions for the config to be updated.
+sleep 120 seconds
```

Comment:
It would be better to wait for the actual session change :grimacing:  - If that is not possible right now, we should add that feature.